packages/FlexGrid/Col/Col.jsx

```diff
@@ -145,12 +167,63 @@ Col.propTypes = {
    * Use the xsOffset prop instead for identical functionality.
    */
   offset: PropTypes.oneOf([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]),
+  /**
+   * Hides the column from the 'xs' breakpoint range.
+   *
+   * `true` hides the column from xs;
+   * `false` shows the column;
+   * By default, the column is visible.
+   *
+   * @since 2.1.0
+   */
+  xsHidden: PropTypes.bool,
```

Comment:
Would it be possible to, instead of creating new props, have this functionality activate if a size prop is set to 0?

For example, `md={0}` would hide the column on the medium breakpoint, rather than setting the new prop `xsMedium` to `true`. That way we can simplify the interface by reducing the amount of required props.